frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -628,6 +628,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedMaxWeight: Get<Weight>;
 
+		/// The maximum amount of unchecked solutions to refund the call fee for. If `None`, all
+		/// solutions, including ejected solutions, will get a refund.
```

Comment:
```suggestion
		/// The maximum amount of unchecked solutions to refund the call fee for.
```

Reply:
TODO: integrity check to make sure this does not go above `T::SignedMaxSubmissions` (although it is harmless, but it clearly shows whoever is configuring the pallet that they've no idea what they are doing)